internal/pkg/store/env.go

```diff
@@ -37,9 +37,7 @@ func (s *Store) CreateEnvironment(environment *archer.Environment) error {
 		if aerr, ok := err.(awserr.Error); ok {
 			switch aerr.Code() {
 			case ssm.ErrCodeParameterAlreadyExists:
-				return &ErrEnvironmentAlreadyExists{
-					EnvironmentName: environment.Name,
-					ProjectName:     environment.Project}
+				return nil
```

Comment:
nit: It'd be great if you can also update `CreateProject` to return `nil` that's the only other `Create*` method that doesn't seem to be returning `nil` if this error happens. (can be in a follow-up PR)